src/hooks/governance-data-provider/useVotingPower.tsx

```diff
@@ -1,45 +1,7 @@
-import { Power } from '@aave/contract-helpers';
-import { gql, useQuery } from '@apollo/client';
-import { useWeb3Context } from 'src/libs/hooks/useWeb3Context';
-
-export type PowerQueryResponseType = {
-  __typename?: 'Query';
-  votingPowers: {
-    __typename?: 'VotingPowers';
-    votingPower: string;
-    propositionPower: string;
-    aaveVotingDelegatee: string;
-    aavePropositionDelegatee: string;
-    stkAaveVotingDelegatee: string;
-    stkAavePropositionDelegatee: string;
-    aaveTokenPower: Power;
-    stkAaveTokenPower: Power;
-  };
-};
-
-export const PowerQuery = gql`
-  query VotingPowers($userAccount: String) {
-    votingPowers(userAccount: $userAccount) {
-      votingPower @client
-      propositionPower @client
-      aaveVotingDelegatee @client
-      aavePropositionDelegatee @client
-      stkAaveVotingDelegatee @client
-      stkAavePropositionDelegatee @client
-      aaveTokenPower @client
-      stkAaveTokenPower @client
-    }
-  }
-`;
+import { useRootStore } from 'src/store/root';
 
 export const useVotingPower = () => {
```

Comment:
This entire hook could just be removed now lol

Reply:
some of these hooks are essentially selectors now and don't really make sense.
That said we kept them as is to not increase the amount of files changed /merge conflicts.
We'll do a follow up pr for sure prettifying some of these things.